data/Store.ts

```diff
@@ -219,7 +219,6 @@ export class Store extends HoistBase {
     _filtered: RecordSet;
 
     private _dataDefaults = null;
-    private _created = Date.now();
```

Comment:
Turns out we read and show this in the instances grid within Hoist Inspector. There are also `_created` member vars on HoistModel and HoistService, although you can tell how essential they are by the fact that the HoistModel one was broken until just now (https://github.com/xh/hoist-react/commit/b0418d03e4982d8d6a6d1b6f9125dfc3f88d6886)

I don't feel strongly about it, but if we keep the timestamp we should not mark as private, obvs, and if we don't think it's useful we might as well remove it from the other two base classes and the little grid in Inspector

Reply:
I'll revert!  Was thrown off by the mistaken use of private.